package.json

```diff
@@ -202,10 +202,10 @@
     "@types/lodash.throttle": "^4.1.7",
     "@types/lodash.uniqby": "^4.7.7",
     "@types/moment": "^2.13.0",
-    "@types/react": "^18.2.55",
-    "@types/react-dom": "^18.2.19",
+    "@types/react": "^19.0.7",
+    "@types/react-dom": "^19.0.3",
     "@types/react-image-gallery": "^1.2.4",
-    "@types/react-is": "^18.2.4",
+    "@types/react-is": "^19.0.0",
```

Comment:
Can we drop depending on `react-is`? I see it is used only in two places. I'll be happy to have one dependency less